### PR TITLE
feat: add bbs signatures example

### PR DESCRIPTION
**Pull request by sairanjit**
### Summary

- Added BBS signatures example

### CODE REVIEW

File: examples.json [hunk 1]
```diff
@@ -28,5 +28,11 @@
 		"name": "Import Did",
 		"path": "examples/import-did",
 		"description": "Import an already existing did, optionally resolving it and importing the associated private keys into the wallet."
+	},
+	{
+		"id": "issue-bbs-credentials",
+		"name": "Issue Bbs Credentials",
+		"path": "examples/issue-bbs-credentials",
+		"description": "Issue a W3C credentials using bbs signatures"
```
Comment (TimoGlastra) [inline examples.json]:
```suggestion
		"description": "Issue a W3C credential using BBS+ signatures"
```

File: examples/issue-bbs-credentials/index.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import {
+	CREDENTIALS_CONTEXT_V1_URL,
+	KeyType,
+	SECURITY_CONTEXT_BBS_URL,
+	TypedArrayEncoder,
+} from "@aries-framework/core";
```
Comment (TimoGlastra) [inline examples/issue-bbs-credentials/index.ts]: Should be credo-ts right?

```suggestion
} from "@credo-ts/core";
```

File: examples/issue-bbs-credentials/index.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+	const credentialOffer = await issuer.credentials.createOffer({
+		credentialFormats: {
+			jsonld: {
+				credential: {
+					"@context": [
+						CREDENTIALS_CONTEXT_V1_URL,
+						"https://www.w3.org/2018/credentials/examples/v1",
+						// Add BBS context to the credential
+						SECURITY_CONTEXT_BBS_URL,
```
Comment (TimoGlastra) [inline examples/issue-bbs-credentials/index.ts:57]: Isn't that done automatically? Might be a nice improvement

File: examples/issue-bbs-credentials/index.ts [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+	// Create out of band invitation
+	const inv = await issuer.oob.createInvitation({
+		messages: [credentialOffer.message],
+	});
+
+	// Accept the invitation
+	const outOfBandRecord = await holder.oob.receiveInvitation(
+		inv.outOfBandInvitation,
+	);
```
Comment (TimoGlastra) [inline examples/issue-bbs-credentials/index.ts]: Should we also accept the credential here? And maybe wait and print the credential that was issued?

Comment (TimoGlastra) [reply examples/issue-bbs-credentials/index.ts]: I think using ContentApproved is better reflective of a real life scenario

File: examples/issue-bbs-credentials/package.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+	"description": "Issue a W3C credentials using bbs signatures"
```
Comment (TimoGlastra) [inline examples/issue-bbs-credentials/package.json]:
```suggestion
	"description": "Issue a W3C credential using BBS+ signatures"
```

**Review by TimoGlastra — APPROVED** [review]
Nice 👍

File: examples/issue-bbs-credentials/index.ts (hunk 3 above)
Comment (sairanjit) [reply examples/issue-bbs-credentials/index.ts:57]: `CREDENTIALS_CONTEXT_V1_URL` doesn't have `BbsBlsSignature2020` so I am using `SECURITY_CONTEXT_BBS_URL`

**Pull request merged**